test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/validate.test.js

```diff
@@ -1405,6 +1405,25 @@ describe('#validate()', () => {
       },
     });
   });
+
+  it('should throw an error when restApiRootResourceId is not provided with restApiId', () => {
```

Comment:
We have a policy that all new tests should be constructed via `runServerless` util (see `describe` below, which already hosts some).

Ideally, we should configure a new `runServerless` based test, that confirms that an error with expected code was thrown